src/ProjectTemplates/BlazorTemplates.Tests/BlazorServerTemplateTest.cs

```diff
@@ -47,21 +57,32 @@ public async Task BlazorServerTemplateWorks_NoAuth()
             var buildResult = await Project.RunDotNetBuildAsync();
             Assert.True(0 == buildResult.ExitCode, ErrorMessages.GetFailedProcessMessage("build", Project, buildResult));
 
+            var browser = Fixture.BrowserManager.IsAvailable(browserKind) ?
+                await Fixture.BrowserManager.GetBrowserInstance(browserKind, BrowserContextInfo) :
+                null;
```

Comment:
This "checking if browser is available or explicitly disabled" logic will be quite a lot to duplicate across every test. Is there any reasonable way this could be factored out into a single place?

Reply:
It's something very specific to these tests because they still run on some platforms when the UI automation is not available. I don't expect in the future to need this logic, since I hope we run automation across platforms, but for now I'm keeping it consistent with what we had.

I don't expect this functionality to be something we need outside of these tests, since for other tests, I expect that we either are able to run the automation or fail